oem/ibm/libpldmresponder/oem_ibm_handler.hpp

```diff
@@ -114,6 +114,7 @@ class Handler : public oem_platform::Handler
                         setEventReceiverCnt = 0;
                         disableWatchDogTimer();
                         pldm::responder::utils::clearLicenseStatus();
+                        pldm::responder::utils::clearDumpSocketWriteStatus();
```

Comment:
Hi Ravi, Host goes to off state sometimes even in the middle of the power-on operation
- like when hostboot does a Reconfig loop.
- when we are doing an MPIPL as well.

Are we sure that in both cases - we are okay to clear the socket state?

Reply:
Manoj

yes we need to clear in both cases, whenever phyp stops running  then reset status flag

Reply:
does  this code hit in reconfigured loop path as well right?

Reply:
no i guess, as its with hostboot